src/neo-vm/OpCode.cs

```diff
@@ -6,8 +6,8 @@ public enum OpCode : byte
         /// <summary>
         /// An empty array of bytes is pushed onto the stack.
         /// </summary>
-        PUSH0 = 0x00,
-        PUSHF = PUSH0,
+        PUSHBYTES0 = 0x00,
+        PUSHF = PUSHBYTES0,
```

Comment:
We can remove `PUSHF`

Reply:
I agree... but let's do that in another PR... I'm reverting all changes not directly connected to the fact that BigInteger 0 needs to be a byte array.